docs/includes/giza.log

```diff
@@ -0,0 +1,21 @@
+WARNING:giza.content.changelog.tasks:changelog generation is not configured.
+WARNING:giza.content.inheritance.apiargs:too many field types returning one at random.
+WARNING:giza.content.inheritance.apiargs:too many field types returning one at random.
+WARNING:giza.content.inheritance.apiargs:too many field types returning one at random.
+WARNING:giza.content.inheritance.apiargs:too many field types returning one at random.
+WARNING:giza.content.inheritance.apiargs:too many field types returning one at random.
+WARNING:giza.content.inheritance.apiargs:too many field types returning one at random.
+WARNING:giza.content.sphinx:source/index.txt:66: WARNING: toctree contains reference to nonexisting document u'.. /getting-started'
+WARNING:giza.content.sphinx:source/includes/apiargs/MongoDBCollection-method-createIndex-option.rst:81: WARNING: Inline literal start-string without end-string.
+WARNING:giza.content.sphinx:source/includes/apiargs/MongoDBCollection-method-createIndex-option.rst:155: WARNING: Inline literal start-string without end-string.
+WARNING:giza.content.sphinx:source/upgrade.txt:None: WARNING: image file not readable: img/save-flowchart.png
```

Comment:
This is an old giza.log, so a lot of these have already been fixed. 

That said, some of the warnings are… shall we say 'erroneous'. We have redoing the `apiargs` type as an OKR this quarter, which will get rid of those warnings that aren't actually warnings. This is also an old log, so the legitimate warnings have already been addressed.